Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -18,12 +18,26 @@
 #    import <UIKit/UIKit.h>
 #    import <objc/runtime.h>
 
+// Instead of using associated objects, we use a global map to store the time to display tracker,
+// spanId, spans in execution, and layout subview spanId to avoid memory issues with associated
```

Comment:
`l`: I think we could change the wording here to state that in an earlier version we used associated objects and explain the problems we had with it.